elrond-wasm-debug/src/meta/meta_generate_snippets.rs

```diff
@@ -0,0 +1,439 @@
+use std::{
+    fs::{self, File},
+    io::Write,
+};
+
+use elrond_wasm::abi::{ContractAbi, EndpointAbi, EndpointMutabilityAbi, InputAbi, OutputAbi};
+
+use super::meta_config::MetaConfig;
+
+impl MetaConfig {
+    // TODO: Handle overwrite flag
```

Comment:
Have an overwrite flag ... we don't want people to accidentally erase their code.